designsafe/static/scripts/projects/components/edit-project/edit-project.component.js

```diff
@@ -186,16 +186,17 @@ class EditProjectCtrl {
                 if (this.project.value.nhEventEnd) {
                     this.form.nhEventDateEnd = new Date(this.project.value.nhEventEnd);
                 }
-                this.form.nhTypes = this.project.value.nhTypes;
+                if (this.project.value.nhTypes.length > 0) {
+                    this.form.nhTypes = this.project.value.nhTypes;
+                } else {
+                    this.form.nhTypes = new Array (1);
```

Comment:
Just curious - any reason this `nhTypes` needs to be an array of size 1, instead of `[]`?

Reply:
We allow the users to add multiple empty input boxes as needed to populate nhTypes. Presetting this with 1 blank field gives us one blank input in the form by default.